samples/client/petstore/python-experimental/pom.xml

```diff
@@ -1,46 +0,0 @@
-<project>
```

Comment:
We need this file for CI tests to work, please add it back

Reply:
Done

Reply:
Can you resolve this conversation?